### ISSUE TITLE
Accessibility - Insufficient Contrast

### ISSUE THREAD

**Issue by dylanhthomas** [nopSolutions/nopCommerce#5024]
nopCommerce version: 4.3

Steps to reproduce the problem:

1. Navigate to homepage of demo site: https://demo.nopcommerce.com/
2. Run a contrast checker, such as WCAG Color Contrast (https://chrome.google.com/webstore/detail/wcag-color-contrast-check/plnahcmalebffmaghcpcmpaciebdhgdf)
3. Navigate to other pages and repeat.
4. Observe the failing styles

It fails WCAG 2.0 - Level AA - Success Criteria - 1.4.3: Contrast (Minimum)

To be WCAG 2.0 AA compliant, all small text needs a contrast ratio of 4.5:1 and all large text needs a contrast ratio of 3:1.

On all pages, 5 styles fail, ranging from 1.9:1 to 4.48:1
On the homepage, 2 additional styles fail.

https://www.w3.org/WAI/WCAG21/quickref/#contrast-minimum
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

**Comment by AndreiMaz** [comment]
Regarding all accessibility tasks. Currently we have "accessiBe" plugin available out of the box.

**Comment by kirilchik** [comment]
After some consideration we've decided not to implement this functionality out of the box and leave it for customization
